README.md

```diff
@@ -1 +1,77 @@
-# Module1-Capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Module1-Capstone
+
+> Module-1 capstone project.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+Here's a [video ](https://www.loom.com/share/4965626eea644be3992bbfe476953354) of me giving a few description of the project.
```

Comment:
- [ ] Kindly make sure to highlight a piece of the code you find interesting or proud of making in the video presentation.